app/src/main/java/com/babylon/wallet/android/presentation/settings/recovery/chooseseed/ChooseSeedPhraseScreen.kt

```diff
@@ -108,6 +108,7 @@ private fun ChooseSeedPhraseContent(
             modifier = Modifier
                 .fillMaxWidth()
                 .padding(RadixTheme.dimensions.paddingDefault).navigationBarsPadding(),
+            enabled = state.isAnyFactorSourceSelected,
```

Comment:
wouldn't be sufficient to just use the existing `selectedFactorSource` and check if it is not null?

Reply:
> wouldn't be sufficient to just use the existing `selectedFactorSource` and check if it is not null?

Hmmm actually you are right I think